test/tools/paths.d

```diff
@@ -0,0 +1,76 @@
+module tools.paths;
+
+import std.file : exists;
+import std.path : buildNormalizedPath, buildPath, dirName, setExtension;
+import std.process : environment;
+
+version (Posix)
+    enum exeExtension = "";
+else version (Windows)
+    enum exeExtension = ".exe";
+
+version(Windows)
```

Comment:
Since `run.d` has been written it was agreed to use a space after `version ` too :man_shrugging: 
At least be consistent in this file.

Reply:
Sure, that's what I prefer too.

Reply:
Fixed.